optimization/ht_functions.py

```diff
@@ -364,12 +347,15 @@ def sweep_geometric_configs(self, diams, pds, z, c):
         # sweep through parameter space, calculate min mass
         for i in range(self.N):
             for j in range(self.N):
-                flowdata = Flow(D_mesh[i, j], PD_mesh[i, j], c, z)
+                if props:
```

Comment:
Probably don't need this conditional if you set up your constructor properly.  If `props == None` then your constructor can invoke the defaults...